### PR TITLE
removed arguements from the get command.

### PR DESCRIPTION
**Pull request by btv**
Potential fix for #6, however I'm sure this change is incomplete. Please let me know if anything would need to be done to get this completed.

### CODE REVIEW

**Comment by gmemstr** [comment]
I think a better approach is what is described in the linked issue - having two different parameters, one for `create` and another for `get` - it would break backwards compatibility, however.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+**NOTE:** This repo is a private fork from:
+[https://github.com/btv/aws-code-deploy-orb](https://github.com/btv/aws-code-deploy-orb)
+which is a fork from:
+[https://github.com/CircleCI-Public/aws-code-deploy-orb](https://github.com/CircleCI-Public/aws-code-deploy-orb)
+
+This is only going to be kept around until the problem is fixed upstream.
```
Comment (gmemstr) [inline README.md]: Should be removed :)

**Review by gmemstr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (btv) [reply README.md]: Yeah, that was a mistake on my part. Will fix.

**Comment by btv** [comment]
@gmemstr I respect the position about coming up with a better fix for this problem, however, it would be nice to have something working immediately. I'm thinking of this PR more as a patch release to at least get current functionality working, which would provide time for the more thorough release at a later date.

**Comment by EricRibeiro** [comment]
Looks like this was included in #11.

**Pull request closed without merging**